packages/keystone/bin/loading.html

```diff
@@ -128,37 +128,44 @@ <h1>Keystone is <span id="status">loading...</span></h1>
 
       function checkAndSetStatus() {
         let cancelled = false;
-        fetch('/', { headers: { Accept: 'application/json' } })
-          .then(result => result.json())
-          .catch(error => {
-            // We can get back an error "Cannot parse JSON" when a HTML
-            // response is returned, so we assume the server is ready
-            onReady();
+        if (!ready) {
+          fetch(window.location.href.replace(window.location.origin, ''), {
+            headers: { Accept: 'application/json' },
           })
-          .then(({ loading, status } = {}) => {
-            if (!loading) {
-              onReady();
-              return;
-            }
-            if (cancelled) {
-              return;
-            }
-            switch (status) {
-              case 'init-keystone': {
-                statusEl.innerHTML = 'initialising...';
-                break;
+            .then(result => {
+              // Will get back HTTP status 425 while still loading
+              if (result.status === 425) {
```

Comment:
Actually check for the status instead of trying to parse JSON (which may not be present once the page 200's)